app/js/checkAnswers.ts

```diff
@@ -140,25 +135,22 @@ function showResults(pageLeft) {
     clearInterval(interval)
     const userAnswers = getUserAnswers()
     checkAnswers(userAnswers).then(function (result) {
-        let percentResult
-        let answered
         if (result.score || result.score === 0) {
             if (pageLeft) {
-                document.querySelector<HTMLElement>('.userMessage').innerHTML = `<h1>Test cancelled!</h1>
+                document.querySelector<HTMLElement>('.greetings').innerHTML = '<p>Test cancelled!</p>'
+                document.querySelector<HTMLElement>('.email_for_results').innerHTML = `
                 <p>This test has ended because you clicked away from the page</p>
-                <p>Please contact the office to discuss further</p>`
+                <p>Please contact the office at <a href="mailto:hello@mayden.academy">hello@mayden.academy</a> to discuss further</p>`
```

Comment:
there seems to be an extra back tic at the end of this line. if not necessary, please remove :)